api/src/modules/google/google.controller.spec.ts

```diff
@@ -1,12 +1,13 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { GoogleController } from './google.controller';
+import { GoogleModule } from './google.module';
 
 describe('GoogleController', () => {
   let controller: GoogleController;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      controllers: [GoogleController],
+      imports: [GoogleModule],
```

Comment:
Is it failing, and is the change intentional? This was part of the boilerplate code 🤔

Reply:
That's weird. Yeah it was failing when `jest` tried to run it. Perhaps related to the module issue though?

Reply:
Yepp! I tested it locally, and this indeed seemed like the Module dependency issue :/ Approved.